disnake/channel.py

```diff
@@ -224,6 +230,14 @@ def permissions_for(self, obj: Union[Member, Role], /) -> Permissions:
     def members(self) -> List[Member]:
         """List[:class:`Member`]: Returns all members that can see this channel."""
         return [m for m in self.guild.members if self.permissions_for(m).read_messages]
+    
+    @property
+    def can_send(self) -> bool:
+        return self.permissions_for(self.guild.me).send_messages
```

Comment:
I think it would be better to also check the `embed_links` permissions:
```python
def can_send(self, with_embeds: bool = True):
    can = self.permissions_for(self.guild.me)
    if with_embeds:
        return can.send_messages and can.embed_links
    return can.send_messages
```

Reply:
There you go, fixed it!